exercises/practice/connect/.meta/config.json

```diff
@@ -0,0 +1,15 @@
+{
+  "authors": [],
```

Comment:
Missing author name here

Reply:
I've added the original author's name

Reply:
It should be your GitHub name if you're the one porting the exercise to this track.

Reply:
Is that required?

Reply:
You can leave it off if you want. You just won’t get the additional rep points on Exercism for authoring the exercise (30 I believe). You’d only get rep for opening this PR.

Reply:
Thanks for the clarification!

Reply:
To be clear, it's your github _username_ that's listed here.